src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -79,6 +79,16 @@ - (void)commonInit {
   _scrollView.showsVerticalScrollIndicator = NO;
   _scrollView.showsHorizontalScrollIndicator = NO;
 
+  NSOperatingSystemVersion iOS9Version = {9, 0, 0};
+  NSProcessInfo *processInfo = [NSProcessInfo processInfo];
+  if ([processInfo respondsToSelector:@selector(isOperatingSystemAtLeastVersion:)] &&
+      [processInfo isOperatingSystemAtLeastVersion:iOS9Version] &&
```

Comment:
We should check for symbol availability instead of doing a version check:

[[UIView class] respondsToSelector:@selector(userInterfaceLayoutDirectionForSemanticContentAttribute:)] &&
[self respondsToSelector:@selector(semanticContentAttribute)]

You can read more about this in Apple's SDK Compatibility Guide:
https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/cross_development/Using/using.html#//apple_ref/doc/uid/20002000-1114537-BABHHJBC